wolfcrypt/src/evp.c

```diff
@@ -3143,6 +3143,69 @@ int wolfSSL_PKCS5_PBKDF2_HMAC(const char *pass, int passlen,
 }
 #endif /* !NO_PWDBASED */
 
+
+#if defined(HAVE_SCRYPT) && defined(HAVE_PBKDF2) && !defined(NO_PWDBASED) && \
```

Comment:
Why `NO_SHA`? script uses SHA256. Perhaps you meant `NO_SHA256`?

Reply:
Yes, replaced it with NO_SHA256. Did same for test_wolfSSL_EVP_PBE_scrypt in tests/api.c